src/utils/isTouchDevice.js

```diff
@@ -1,4 +1,4 @@
 export default function isTouchDevice() {
-  return !!(typeof window !== 'undefined' && 'ontouchstart' in window) ||
+  return !!(typeof window !== 'undefined' && window.ontouchstart != null) ||
```

Comment:
By looking at what Modernizr [does](https://github.com/Modernizr/Modernizr/blob/5eea7e2a213edc9e83a47b6414d0250468d83471/feature-detects/touchevents.js), it looks like they do some more checks too. Even though it is not still a bug I could still update this to have a better check.